Dockerfile

```diff
@@ -234,40 +236,47 @@ RUN --mount=type=cache,target=/root/.cache/cmake,id=cmake-ngen \
           -DNGEN_WITH_TESTS=ON \
           -DNGEN_WITH_ROUTING=ON \
           -DNGEN_QUIET=ON \
-          -DNGEN_UPDATE_GIT_SUBMODULES=OFF \
+          -DNGEN_UPDATE_GIT_SUBMODULES=ON \
           -DBOOST_ROOT=/opt/boost && \
-      cmake --build cmake_build --target all
+      cmake --build cmake_build --target all && \
+      rm -rf cmake_build
 
 # Build each submodule in a separate layer, using cache for CMake as well
 RUN --mount=type=cache,target=/root/.cache/cmake,id=cmake-lasam \
     set -eux && \
     cmake -B extern/LASAM/cmake_build -S extern/LASAM/ -DNGEN=ON && \
-    cmake --build extern/LASAM/cmake_build/
+    cmake --build extern/LASAM/cmake_build/ && \
+    rm -rf extern/LASAM/cmake_build/
```

Comment:
All of these BMI model builds seem to be turned into expensive no-ops, since there's nothing from the actual build being copied or otherwise captured.

Is there some plan/intention to copy/install the build `libbmifoo.so` files somewhere else, so that the build trees actually can be deleted?